components/docs-chef-io/data/automate/api-static/02-meta-description.yaml

```diff
@@ -41,3 +41,17 @@ info:
     The Chef Automate documentation covers policies, authentication, and authorization in greater detail.
     * **[API Tokens](https://automate.chef.io/docs/api-tokens/)** explains authentication in Chef Automate.
     * **[Authorization Overview](https://automate.chef.io/docs/iam-v2-overview)** explains authorization in Chef Automate.
+
+    # Status Codes
+
+    | Status Code | Name                  | Description                          | Remark                                                                                                                                                                                                                                                      |
+    |-------------|-----------------------|--------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+    | 200         | OK                    | Correct Id, Auth token and JSON body | The API worked as expected                                                                                                                                                                                                                                  |
+    | 400         | Bad Request           | Wrong JSON body                      | The JSON body contains a key or value missing or in some cases when there is no json.  In some APIs like `IAM Project Rules`, the API request body validation is given the highest preference. And will return this error even before the Auth token validation. |
```

Comment:
Remark: The JSON body contains a key or value missing, sometimes when there is no .json file. In some APIs like `IAM Project Rules`, the API request body validation is given the highest preference. And will return this error even before the Auth token validation.